nixos/modules/services/hardware/intel-aesmd.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, utils, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.intel-aesmd;
+
+in {
+
+  options.services.intel-aesmd = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable Intel's Architectural Enclave Service Manager for Intel SGX
+      '';
+    };
+  };
+
+  options.services.intel-aesmd.package = mkOption {
+    type = types.package;
+    default = pkgs.intel-aesmd;
+    defaultText = "pkgs.intel-aesmd";
+    description = ''
+      aesmd derivation to use.
+    '';
+  };
+
+  config = mkIf cfg.enable {
+    users.groups.aesmd = {
+    };
+
+    users.users.aesmd = {
+      description = "aesmd user";
+    };
+
+    environment.systemPackages = [ cfg.package ];
```

Comment:
Does this package have some use in the user environment? If not this can be removed

Reply:
Removed